### PR TITLE
[GuestConfiguration]Add version field to ConfigurationInfo

### PR DESCRIPTION
**Pull request by aashish1987**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] Please add REST spec PR link to the SDK PR
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by aashish1987** [comment]
https://github.com/Azure/azure-rest-api-specs/pull/5131

**Review by dsgouda — CHANGES_REQUESTED** [review]
@aashish1987 please regenerate the code using generate.ps1 and update the PR.
Also, please bump nuget version to `1.1.0` in the csproj and AssemblyInfo.cs and update the PackageReleaseNotes accordingly

File: src/SDKs/GuestConfiguration/Management.GuestConfiguration/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure GuestConfiguration Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Guest Configuration management operations including the ability to create, update and get guest configuration assignments, compliance statuses, guest configuration assignment reports.")]
 
-[assembly: AssemblyVersion("1.0.0")]
-[assembly: AssemblyFileVersion("1.0.0")]
+[assembly: AssemblyVersion("1.1.0")]
```
Comment (dsgouda) [inline src/SDKs/GuestConfiguration/Management.GuestConfiguration/Properties/AssemblyInfo.cs]: `AssemblyVersion` remains the same, only `AssemblyFileVersion` must be updated

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks good apart from a minor change

**Review by dsgouda — APPROVED** [review]
LGTM will merge once CIs pass

**Pull request merged**